kadai-adapter-kadai-connector/src/main/java/io/kadai/adapter/kadaiconnector/api/impl/KadaiSystemConnectorImpl.java

```diff
@@ -40,29 +39,29 @@
 import java.util.Map;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
-import javax.sql.DataSource;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
 /** Implements KadaiConnector. */
 @Component
 public class KadaiSystemConnectorImpl implements KadaiConnector {
 
   static final String REFERENCED_TASK_ID = "referenced_task_id";
-  static final String REFERENCED_TASK_VARIABLES = "referenced_task_variables";
   static final String SYSTEM_URL = "system_url";
   private static final String TASK_STATE_CANCELLED = "CANCELLED";
   private static final String TASK_STATE_TERMINATED = "TERMINATED";
   private static final Logger LOGGER = LoggerFactory.getLogger(KadaiSystemConnectorImpl.class);
 
-  @Autowired private TaskService taskService;
-  @Autowired private KadaiEngine kadaiEngine;
+  private final TaskService taskService;
+  private final TaskInformationMapper taskInformationMapper;
 
-  @Autowired private TaskInformationMapper taskInformationMapper;
+  public KadaiSystemConnectorImpl(
```

Comment:
Where is the constructor injection for kadaiEngine and kadaiDataSource?

Reply:
I deleted them because they were not used :)